style.css

```diff
@@ -0,0 +1,108 @@
+body {
+    background-color: #fff;
+    width: 100%;
+    font-family: 'Proxima Nova', Georgia, sans-serif;
+    margin: 0 auto;
+}
+
+
+header {
+    max-width: 100%;
+    height: 567px;
+    background-image: url("images/header-bg.jpg");
+    background-repeat: no-repeat;
+    background-position: top center;
+    background-size: cover;
+    color: #fff;
+    padding: 38px 10px 20px;
+    position: relative;
+}
+
+header img {
+    display: block;
+    max-width: 100%;
+    margin: 0 auto 140px;
+}
+
+header h1 {
+    position: relative;
+    top: -6%;
+    text-transform: uppercase;
+    font-size: 2em;
+    text-align: left;
+}
+
+.second_h1 {
+    position: relative;
+    left: 10%;
+}
+
+.center-line {
+    position: relative;
+    top: 10%;
+    left: 20%;
+    border: 0;
+    width: 225px;
+    height: 4px;
+    background-color: #5d40ad;
+    transform: rotate(135deg);
+    margin: 0;
+}
+
+.button-load {
+    background-color: #5d40ad;
+    width: 32%;
+    border-radius: 20px;
+    margin: 85px auto 0;
+    text-transform: uppercase;
+    font-size: 0.5em;
+    padding: 10px 10px 10px 40px;
+}
+
+.app-store {
+    display: block;
+    float: left;
+    background-image: url(images/Icon-Apple.jpg);
+    background-repeat: no-repeat;
+    background-position: left 20px center;
+    text-align: left;
+}
+
+.google-play {
+    display: block;
+    float: right;
+    background-image: url(images/Icon-Apple.jpg);
+    background-repeat: no-repeat;
+    background-position: left 20px center;
+    text-align: left;
+}
+
+nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    flex-wrap: wrap;
+    clear: both;
+    padding: 45px 0 0;
+}
+nav a{
+    display: inline-block;
+    color: #5d40ad;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 0.75em;
+    margin: 10px;
+    text-align: center;
+}
+
+
+
+
+@media only screen and (min-width: 321px) {
```

Comment:
should be something like
`@media all and (min-width: 320px)`
As there are devices with 320px screen resolution